tests/cases/fourslash/completionsRecommended_union.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path="fourslash.ts" />
+
+// @strictNullChecks: true
+
+////const enum E { A = "A", B = "B" }
+////const enum E2 { X = "X", Y = "Y" }
+////const e: E | undefined = /*a*/
+////const e2: E | E2 = /*b*/
+
+verify.completions(
+    {
+        marker: "a",
+        includes: { name: "E", isRecommended: true },
+        isNewIdentifierLocation: true,
+    },
+    {
+        marker: "b",
+        // Arbitrarily chooses one to be recommended
+        includes: [{ name: "E", isRecommended: true, }, { name: "E2" }],
```

Comment:
Why not E2 to be recommended as well?

Reply:
In the original design (#20020) there can only be one recommended completion -- editors may be expecting this. We could add more than one but it would mean more work for editors and I think that's why we avoided it.